platform/lib/samples/examples/.gitignore

```diff
@@ -0,0 +1 @@
+dist
```

Comment:
Looks like everything in this folder is not used.

Reply:
removed